src/components/Composer/index.android.tsx

```diff
@@ -57,7 +24,7 @@ function Composer(
     }: ComposerProps,
     ref: ForwardedRef<TextInput>,
 ) {
-    const textInput = useRef<TextInput>();
+    const textInput = useRef<TextInput | null>(null);
```

Comment:
```suggestion
    const textInput = useRef<TextInput>(null);
```

I think you don't need to specify `null` type.

Reply:
If I remove `null`, it is going to show the following error on **line 33**:

`Cannot assign to 'current' because it is a read-only property.`